requirements.txt

```diff
@@ -1,44 +1,44 @@
-gunicorn==22.0.0
+gunicorn==23.0.0
 
-Django==4.2.20
-whitenoise==6.6.0
+Django==4.2.23
+whitenoise==6.9.0
 
 django-cron==0.6.0 #deprecated
 
 django-watchman==1.3.0
 django-su==1.0.0
-django-mysql==4.12.0
+django-mysql==4.17.0
 # The alternative to this is redis, but we don't use redis yet
-django-constance[database]==3.1.0
-django-webpack-loader==2.0.1
+django-constance[database]==4.3.2
+django-webpack-loader==3.1.1
 
-django-csp==3.7  #no updates
-django-import-export==3.3.1
-django-admin-rangefilter==0.11.2
+django-csp==4.0
+django-import-export==4.3.7
+django-admin-rangefilter==0.13.2
 # Django/Python fontawesome library supporting CSS in UI
-fontawesomefree==6.4.2
+fontawesomefree==6.6.0
 
 # graphql
-graphene-django==3.1.5
-graphql-core-promise==3.2.3.post1
-django-filter==23.3
+graphene-django==3.2.3
+graphql-core-promise==3.4.2
+django-filter==25.1
 
 # object-level permissions
-rules==3.3
+rules==3.5
 
 # These should be okay to update minors
-numpy==1.26.1
-pandas==2.1.2
+numpy==2.2.5
+pandas==2.3.0
 pangres==4.2.1
 
-SQLAlchemy==2.0.23
-mysqlclient==2.2.4
-google-cloud-bigquery[pandas]==3.19.0
```

Comment:
I see this warning when running the cron. 

```
/usr/local/lib/python3.13/site-packages/google/cloud/bigquery/table.py:1957: UserWarning: BigQuery Storage module not found, fetch data with the REST endpoint instead.
  warnings.warn(
```

Reply:
after running the cron, I don't see the data in resources or grades view. is there something I need to configure w.r.t to `courses` table so that I pull data.  I am pretty sure before running cron I had data for the course I was testing.

I don't think the above statement could  be related.
Here is how my course table data looks like
<img width="1572" height="171" alt="Screenshot 2025-07-29 at 3 06 16 PM" src="https://github.com/user-attachments/assets/726fcd2f-d124-4ebf-8eda-8cf044061505" />

@jonespm thoughts

Reply:
Yeah, not sure what version it happened in but looks like we have to do

`google-cloud-bigquery[pandas, bqstorage]==3.34.0 now`